src/app/public/modules/radio/radio-group.component.spec.ts

```diff
@@ -115,35 +120,14 @@ describe('Radio group component', function () {
     expect(componentInstance.radioGroupComponent.value.name).toEqual('Lillith Corharvest');
 
     componentInstance.radioForm.patchValue({
-      option: componentInstance.options[1]
+      radioGroup: componentInstance.options[1]
     });
     fixture.detectChanges();
     tick();
 
     expect(componentInstance.radioGroupComponent.value.name).toEqual('Harima Kenji');
   }));
 
-  it('should handle disabled state properly', fakeAsync(function (done: Function) {
```

Comment:
This test was covering radio component logic and NOT radio group logic. I double checked the `radio.component.spec.ts` file and saw we have a duplicate test there. Coverage did not drop either. Removed!